package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "dataDashboard",
+  "description": "An application for rapidly prototyping accessible data graphics.",
+  "version": "0.1",
+  "author": "Michael Skirpan",
+  "bugs": "https://issues.fluidproject.org",
+  "homepage": "http://www.fluidproject.org/",
+  "dependencies": {},
+  "licenses": [
+    {
+      "type": "BSD-3-Clause",
+      "url": "http://www.opensource.org/licenses/BSD-3-Clause"
+    },
+    {
+      "type": "ECL-2.0",
+      "url": "http://www.opensource.org/licenses/ECL-2.0"
+    }
+  ],
+  "keywords": [
+    "visualization",
+    "graphs",
+    "data",
+    "fluid",
+    "infusion",
+    "sonfication"
+  ],
+  "repository": "git://github.com/fluid-project/data-dashboard.git",
+  "main": "",
+  "engines": {
+    "node": ">=0.10.0"
+  },
```

Comment:
Could you flesh out this package.json so that it is used to source the dependencies for the project from npm - infusion, d3, and PapaParse, and update to the latest packaging format and version of dependencies - you can consult https://github.com/fluid-project/kettle/blob/master/package.json#L15 for comparison

The project name should be a valid npm name, which means it can't include capital letters. I suggest something like fluid-data-dashboard or so 

After you've moved over to npm, you'll need to update all your dependency paths from lib/ to node_modules etc.

Please update Gruntfile.js in step also